aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -237,20 +241,28 @@ def lambda_handler(event, context):
     current_event: BaseProxyEvent
     lambda_context: LambdaContext
 
-    def __init__(self, proxy_type: Enum = ProxyEventType.APIGatewayProxyEvent, cors: CORSConfig = None):
+    def __init__(
+        self,
+        proxy_type: Enum = ProxyEventType.APIGatewayProxyEvent,
+        cors: CORSConfig = None,
+        debug: Optional[bool] = None,
+    ):
         """
         Parameters
         ----------
         proxy_type: ProxyEventType
             Proxy request type, defaults to API Gateway V1
         cors: CORSConfig
             Optionally configure and enabled CORS. Not each route will need to have to cors=True
+        debug: Optional[bool]
+            Enables debug mode, by default False. Can be enabled by "POWERTOOLS_API_DEBUG" environment variable
```

Comment:
Could you change the env var to `POWERTOOLS_EVENT_HANDLER_DEBUG`?

This way we remain consistent with `POWERTOOLS_<utility>_<feature>`. This also means we could reuse in AppSync Event Handler or future ones